### ISSUE TITLE
Failed to create Conda environment.

### ISSUE THREAD

**Issue by linette-zyy** [microsoft/PTVS#6329]
**Environment**
![image](https://user-images.githubusercontent.com/59192491/102859478-e9839600-4466-11eb-8083-4929f2431730.png)

**Steps to Reproduce**
1. Create python application
2. Right-click in python environment node and select add environment
3. Select Conda environment tab, select like below:
![image](https://user-images.githubusercontent.com/59192491/102860070-15ebe200-4468-11eb-9879-b3c3975cb834.png)
4. Click "Create" button

**Expected behavior**
Conda virtual environment is created successfully

**Actual result**
Failed to create Conda environment.
![image](https://user-images.githubusercontent.com/59192491/102860305-85fa6800-4468-11eb-83c4-0049564c906e.png)

**Comment by linette-zyy** [comment]
https://github.com/Microsoft/PTVS/issues/5020
From the above issue, this issue should be fixed in version 4.5.10，My conda version is 4.5.11，I am not sure if this problem is solved on this version.

**Comment by bschnurr** [comment]
On 4.9.2 it works 

```
Refreshing interpreters
Refreshed interpreters
Creating new analyzer
Waiting for existing call
Created analyzer <VsProjectAnalyzer:Global|PythonCore|3.7:<Process 25420>:C:\Users\bschnurr\source\repos\PythonApplication52\PythonApplication52.pyproj>
Project includes files:
    C:\Users\bschnurr\source\repos\PythonApplication52\PythonApplication52.py
Setting search paths
Adding 1 files
Existing call complete
----- Creating 'env2' -----
Collecting package metadata (current_repodata.json): ...working... done
Solving environment: ...working... done
## Package Plan ##
  environment location: C:\Users\bschnurr\Anaconda3\envs\env2
  added / updated specs:
    - python=3.7
The following packages will be downloaded:
    package                    |            build
    ---------------------------|-----------------
    ca-certificates-2020.12.8  |       haa95532_0         122 KB
    certifi-2020.12.5          |   py37haa95532_0         141 KB
    openssl-1.1.1i             |       h2bbff1b_0         4.8 MB
    pip-20.3.3                 |   py37haa95532_0         1.8 MB
    python-3.7.9               |       h60c2a47_0        14.4 MB
    setuptools-51.0.0          |   py37haa95532_2         726 KB
    vc-14.2                    |       h21ff451_1           8 KB
    vs2015_runtime-14.27.29016 |       h5e58377_2        1007 KB
    wheel-0.36.2               |     pyhd3eb1b0_0          33 KB
    ------------------------------------------------------------
                                           Total:        23.0 MB
The following NEW packages will be INSTALLED:
  ca-certificates    pkgs/main/win-64::ca-certificates-2020.12.8-haa95532_0
  certifi            pkgs/main/win-64::certifi-2020.12.5-py37haa95532_0
  openssl            pkgs/main/win-64::openssl-1.1.1i-h2bbff1b_0
  pip                pkgs/main/win-64::pip-20.3.3-py37haa95532_0
  python             pkgs/main/win-64::python-3.7.9-h60c2a47_0
  setuptools         pkgs/main/win-64::setuptools-51.0.0-py37haa95532_2
  sqlite             pkgs/main/win-64::sqlite-3.33.0-h2a8f88b_0
  vc                 pkgs/main/win-64::vc-14.2-h21ff451_1
  vs2015_runtime     pkgs/main/win-64::vs2015_runtime-14.27.29016-h5e58377_2
  wheel              pkgs/main/noarch::wheel-0.36.2-pyhd3eb1b0_0
  wincertstore       pkgs/main/win-64::wincertstore-0.2-py37_0
  zlib               pkgs/main/win-64::zlib-1.2.11-h62dcd97_4
Downloading and Extracting Packages
wheel-0.36.2         | 33 KB     |            |   0% 
wheel-0.36.2         | 33 KB     | ########## | 100% 
wheel-0.36.2         | 33 KB     | ########## | 100% 
openssl-1.1.1i       | 4.8 MB    |            |   0% 
openssl-1.1.1i       | 4.8 MB    | #4         |  15% 
openssl-1.1.1i       | 4.8 MB    | ##7        |  27% 
openssl-1.1.1i       | 4.8 MB    | ####6      |  47% 
openssl-1.1.1i       | 4.8 MB    | #######1   |  71% 
openssl-1.1.1i       | 4.8 MB    | ########## | 100% 
openssl-1.1.1i       | 4.8 MB    | ########## | 100% 
vs2015_runtime-14.27 | 1007 KB   |            |   0% 
vs2015_runtime-14.27 | 1007 KB   | #####7     |  57% 
vs2015_runtime-14.27 | 1007 KB   | ########## | 100% 
vs2015_runtime-14.27 | 1007 KB   | ########## | 100% 
vc-14.2              | 8 KB      |            |   0% 
vc-14.2              | 8 KB      | ########## | 100% 
ca-certificates-2020 | 122 KB    |            |   0% 
ca-certificates-2020 | 122 KB    | ########## | 100% 
ca-certificates-2020 | 122 KB    | ########## | 100% 
pip-20.3.3           | 1.8 MB    |            |   0% 
pip-20.3.3           | 1.8 MB    | ##3        |  23% 
pip-20.3.3           | 1.8 MB    | #######2   |  73% 
pip-20.3.3           | 1.8 MB    | ########## | 100% 
pip-20.3.3           | 1.8 MB    | ########## | 100% 
certifi-2020.12.5    | 141 KB    |            |   0% 
certifi-2020.12.5    | 141 KB    | ########## | 100% 
certifi-2020.12.5    | 141 KB    | ########## | 100% 
setuptools-51.0.0    | 726 KB    |            |   0% 
setuptools-51.0.0    | 726 KB    | ########8  |  88% 
setuptools-51.0.0    | 726 KB    | ########## | 100% 
python-3.7.9         | 14.4 MB   |            |   0% 
python-3.7.9         | 14.4 MB   | 4          |   4% 
python-3.7.9         | 14.4 MB   | #1         |  11% 
python-3.7.9         | 14.4 MB   | #8         |  18% 
python-3.7.9         | 14.4 MB   | ##5        |  26% 
python-3.7.9         | 14.4 MB   | ###2       |  33% 
python-3.7.9         | 14.4 MB   | ###8       |  38% 
python-3.7.9         | 14.4 MB   | ####7      |  47% 
python-3.7.9         | 14.4 MB   | #####4     |  54% 
python-3.7.9         | 14.4 MB   | ######1    |  61% 
python-3.7.9         | 14.4 MB   | ######8    |  69% 
python-3.7.9         | 14.4 MB   | #######5   |  76% 
python-3.7.9         | 14.4 MB   | ########2  |  83% 
python-3.7.9         | 14.4 MB   | #########  |  90% 
python-3.7.9         | 14.4 MB   | #########7 |  97% 
python-3.7.9         | 14.4 MB   | ########## | 100% 
Preparing transaction: ...working... done
Verifying transaction: ...working... done
Executing transaction: ...working... done
#
# To activate this environment, use
#
#     $ conda activate env2
#
# To deactivate an active environment, use
#
#     $ conda deactivate
----- Successfully created 'env2' -----

Refreshing interpreters
Refreshed interpreters
Unhooking events from <VsProjectAnalyzer:Global|PythonCore|3.7:<Process 25420>:C:\Users\bschnurr\source\repos\PythonApplication52\PythonApplication52.pyproj>
Unhooked events
Creating new analyzer
Created analyzer <VsProjectAnalyzer:CondaEnv|CondaEnv|env2:<Process 17760>:C:\Users\bschnurr\source\repos\PythonApplication52\PythonApplication52.pyproj>
Transferring from old analyzer <VsProjectAnalyzer:Global|PythonCore|3.7:<Process 25420>:C:\Users\bschnurr\source\repos\PythonApplication52\PythonApplication52.pyproj>, which has 1 files
Tranferred 1 files
Old analyzer now has 0 files
Disposing old analyzer
Project includes files:
    C:\Users\bschnurr\source\repos\PythonApplication52\PythonApplication52.py
Setting search paths
Adding 1 files

```
